set-backend-deployment-env.sh

```diff
@@ -0,0 +1,32 @@
+
+set -o allexport
+[[ -f tdrs-backend/.env.production ]] && source tdrs-backend/.env.production
```

Comment:
Does this file get created by another function is this PR?

Additionally, any reason you didn't go with specifying the environment variables in the manifest? I think we'll need to do it that way to handle the differing variables used for dev and staging.